Dockerfile

```diff
@@ -15,17 +25,27 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-RUN pip install --upgrade pip
-RUN conda update setuptools
+RUN chown -R ${NB_UID} /opt/conda/
+
+COPY requirements.txt ${HOME}/
+
+USER ${NB_USER}
+
+RUN pip install --upgrade pip && \
+      pip install -r ${HOME}/requirements.txt && \
+      conda update setuptools && \
+      conda install jupyter notebook scikit-learn
```

Comment:
This should be elsewhere because not necessarily people will use Docker. Why not `requirements.txt`? Or maybe creating a `conta_requirements.txt` file.

Reply:
It was there before me, so I thought it was right. Now I moved to `requirements.txt`